track/mounts.py

```diff
@@ -141,6 +141,19 @@ def safe(self) -> bool:
         """
 
 
+    @abstractmethod
+    def shutdown(self) -> bool:
+        """Bring mount into a safe state and shut down.
+
+        This method is expected to perform the same actions as `safe()` but also perform any
+        additional actions needed to prepare for program end. After this method is called any calls
+        to other methods in this class may fail.
+
+        Returns:
+            True if the mount was safed successfully. False otherwise.
+        """
+
+
```

Comment:
- [ ] Changes in this file should be a separate commit since they are mostly unrelated to guidescope alignment automation. (Check for calls to `shutdown()` in other source files and bring those over too.)